rep-0003.rst

```diff
@@ -220,18 +216,64 @@ Exact or Series Requirements:
 - Qt 5.3.x
 - PyQt5
 
-Buildsystem support:
+Build System Support:
 
-- catkin:
+- Same as Indigo
 
-  - build from source
-  - release for binary packaging
-  - wiki documentation
-  - continuous integration
+Lunar Loggerhead (May 2017 - May 2019)
+--------------------------------------
+Required Support for:
 
-- rosbuild:
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
 
-  - build from source
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 26
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Minimum Requirements:
+
+ Most of the following requirements are the same as Kinetic's to avoid maintainers
+ branching out before the next LTS release
+
+- C++11
+- Python 2.7
+
+  - Python 3.5 not required, but testing against it is recommended
+
+- Lisp SBCL 1.2.4
+- CMake 3.0.2
+- Boost 1.55
+
+Exact or Series Requirements:
+
+- Ogre3D 1.9.x
+- Gazebo 8 (packaged by OSRF)
+
+  - ROS won't use upstream Gazebo for the upcoming few releases because of a mismatch
+    between upstream versions on targeted Ubuntu distributions
+  - In the case of Lunar:
+
+    - Zesty ships with Gazebo 8
+    - Yakkety and Xenial ship with Gazebo 7
+
+- PCL 1.7.x (Xenial)
+- PCL 1.8.x (Other Systems)
```

Comment:
I didn't update the PCL PPA anymore because PCL is integrated into Ubuntu. I tried to compile 1.8 for Trusty some time ago, but ran into compiler errors (I guess due to memory limitations on the PPA servers):
https://launchpad.net/~v-launchpad-jochen-sprickerhof-de/+archive/ubuntu/pcl18/+packages

Having 1.8 in Xenial would surely be nice, but note that this would mean you would develop against a different version then other Xenial systems have. As long as you don't want to link in other code compiled on Xelinal (against 1.7), this is fine (I'm not aware of anything like this). Also runtime wise it should be fine as you can install both at the same time due ABI versioning.